starwars/src/components/Character.js

```diff
@@ -0,0 +1,28 @@
+import React from "react";
+
+function Character(props) {
```

Comment:
You can also do some destructuring:

`const {height, mass, hair_color, birth_year, gender} = props.char`

`<p><strong>Height:</strong> {height}</p>`

It's less verbose